pages/linux/select.md

```diff
@@ -7,9 +7,13 @@
 
 `select {{word}} in {{apple}} {{orange}} {{pear}} {{banana}}; do echo ${{word}}; done`
 
-- Create a menu for picking a file or folder from the current directory:
+- Create a menu from the output of another program:
 
-`select {{file}} in *; do echo ${{file}}; done`
+`select {{line}} in $({{program_to_run}}); do echo ${{line}}; done`
+
+- Set the prompt string for select and create a menu for picking a file or folder from the current directory:
```

Comment:
It's better to use set to somewhat permanent changes and specify/use/with for temporary changes
```suggestion
- Specify the prompt string for select and create a menu for picking a file or folder from the current directory:
```
Or:
```suggestion
- Create a menu and prompt with a specific string for a file or folder from the current directory:
```

Reply:
Since the prompt string is a different operation, I'll pick the first suggestion. This is so that the language follows the chronological order of what is happening.